### PR TITLE
add a dummy file-launcher

### PR DESCRIPTION
**Pull request by TheOrangeCat**

### CODE REVIEW

**Comment by protonesso** [comment]
That's not like "Slackware-ish". Anyway `kagami` binary will exist when I finish development of `craftpkg` (sorry in private repo). Also your stuff can't handle more that 1 argument. If I don't want to run post install script, I probably enter this in my terminal: `kagami installpkg -ns librsvg#2.40.10-1.bhv.ppc.tar.zst` but it will give me an error!

**Comment by protonesso** [comment]
Also kagami is a out emerge or prt-get

**Comment by protonesso** [comment]
our, not "out"

File: kagami [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
-removepkg $2
+removepkg $2 $3 $4 $5 $6
 ;;
 craftpkg)
-craftpkg $2
+craftpkg $2 $3 $4 $5 $6
 ;;
 *)
-echo "Usage: kagami {installpkg|removepkg|craftpkg} package"
+echo "Usage: kagami {installpkg|removepkg|craftpkg} [options]"
```
Comment (protonesso) [inline kagami]: i think that's not good idea

Comment (TheOrangeCat) [reply kagami]: ok

Comment (TheOrangeCat) [reply kagami]: just if i say package thats also not a good idea

Comment (protonesso) [reply kagami]: I meant $3 $4 $5

Comment (TheOrangeCat) [reply kagami]: theres no way to assign to make it based on how many options there are

Comment (TheOrangeCat) [reply kagami]: $# is useless

Comment (TheOrangeCat) [reply kagami]: cuz you cant make $$i

**Comment by protonesso** [comment]
dude I can do this script on my own. I have another representation of kagami. kagami is a source-based package manager.

**Comment by protonesso** [comment]
installpkg, removepkg and craftpkg are backends of kagami. I don't want to make a mess in one script

**Pull request closed without merging**